image/templates/helm/stackrox-secured-cluster/templates/collector.yaml.htpl

```diff
@@ -131,6 +131,12 @@ spec:
               resource: limits.cpu
         - name: ROX_CALL_NODE_INVENTORY_ENABLED
           value: {{ if eq ._rox.env.openshift 4 }}"true"{{ else }}"false"{{ end }}
+        - name: ROX_NODE_INDEX_ENABLED
+          value: {{ if eq ._rox.env.openshift 4 }}"true"{{ else }}"false"{{ end }}
```

Comment:
I see that two lines above we are using essentially the same thing, but I am thinking, wouldn't something like

```
        - name: ROX_NODE_INDEX_ENABLED
          value: {{ eq ._rox.env.openshift 4 | quote }}
```

be an improvement for readers?

Reply:
I'm fine either way.
Personally, I find the more explicit version better readable, but on the other hand, I don't work too much with Helm charts. Happy to change to the shorter version to improve readability